scRNA-seq-advanced/setup/mito_gene_lists.Rmd

```diff
@@ -0,0 +1,105 @@
+---
+title: "Mitochondrial Gene Lists"
+author: Joshua Shapiro for the CCDL
+date: 2022-11
+output: 
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+This notebook generates a list of mitochondrial genes for use in scRNA-seq quality control.
+
+Most of the functionality of this notebook is accomplished via Bioconductor annotations. For more information about using Bioconductor annotation packages, see the
+[AnnotationDbi package vignette](http://bioconductor.org/packages/release/bioc/vignettes/AnnotationDbi/inst/doc/IntroToAnnotationPackages.pdf).
+
+To get current annotations, we will use [AnnotationHub](https://bioconductor.org/packages/release/bioc/html/AnnotationHub.html).
+
+## Setup
+
+```{r setup}
+# magrittr for pipes
+library(magrittr)
+```
+
+
+First some locations for the gene lists we will be making. These will be in the /shared directory
+
+```{r paths}
+data_dir = "/shared/data/training-modules/scRNA-seq-advanced/data"
+
+# the human gene list will be in the reference subfolder.
+hs_mito_file <- file.path(data_dir, "reference", "hs_mitochondrial_genes.tsv")
+
+```
+
+
+## Human Ensembl Annotation
+
+### Retrieving the db
+
+To get the annotations, we will first create an `AnnotationHub` object, which is used as the interface to the annotation files that the Bioconductor project makes available.
+We then query it for to find out what Ensdb packages (containing Ensembl annotations) are available for *Homo sapiens*.
+
+```{r}
+hub <- AnnotationHub::AnnotationHub()
+```
+
+```{r}
+AnnotationHub::query(hub, c("Ensdb", "Homo sapiens"))
+```
+We are interested in the annotation package for Ensembl 95, which was what we used for the the annotations we built this time around.
```

Comment:
```suggestion
We are interested in the annotation package for Ensembl 95, which was what we used for the annotations we built this time around.
```

Reply:
Confirming we still want 95?

Reply:
I wasn't 100% sure, but I don't think it really matters for this; the mitochondrial genes haven't changed in a while, as far as I know!